src/query/functions/src/scalars/comparisons/comparison_like.rs

```diff
@@ -76,10 +76,25 @@ impl StringSearchImpl for StringSearchLike {
             PatternType::PatternStr => {
                 let pattern = simdutf8::basic::from_utf8(rhs)
                     .expect("Unable to convert the LIKE pattern to string: {}");
+                let mut sub_strings: Vec<&str> =
```

Comment:
Other characters in regexp should be taked care:

```
*,+,{},?
```

Reply:
Yep, that's a bug:

```sql
mysql> select * from t;
+------+
| id   |
+------+
| +    |
+------+
1 row in set (0.04 sec)
Read 1 rows, 17.00 B in 0.014 sec., 70.29 rows/sec., 1.17 KiB/sec.

mysql> select * from t where id like '%+%';
Empty set (0.03 sec)
Read 1 rows, 17.00 B in 0.018 sec., 56.15 rows/sec., 954.56 B/sec.

```